android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -228,10 +228,14 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
             default:
         }
 
-        if(session.isBookmarked()) {
+        if(session.isBookmarked() && (Color.parseColor(session.getTrack().getFontColor()) == Color.WHITE)) {
```

Comment:
fine then.
also should i do the same for search icon and app bar title? or should i leave them for now.